### DISCUSSION TITLE
How to customize the colors of disabled buttons?

### DISCUSSION THREAD

**Discussion by jaimeiniesta** [saadeghi/daisyui#2812] [Q&A] [answered]
I'd like to customize the colors of `btn-disabled`, which out of the box look like this and are hard to read:

![Captura de pantalla 2024-02-02 a las 13 27 27](https://github.com/saadeghi/daisyui/assets/2629/f49fe7d9-2b28-4b34-ac03-73933472344d)

However, according to https://daisyui.com/docs/colors/ it's not possible to customize that?

**Comment by mark1502** [comment]
Not sure, but try something like disabled:bg-blue-600 in the class.

**Reply by jaimeiniesta** [reply]
Thanks, but I mean customizing it from the Daisy UI config just like the other colors `primary`, `secondary`, `error`, etc., can be customized.

**Comment by himanshujangid03** [comment]
> I'd like to customize the colors of `btn-disabled`, which out of the box look like this and are hard to read:
> 
> ![Captura de pantalla 2024-02-02 a las 13 27 27](https://private-user-images.githubusercontent.com/2629/301842961-f49fe7d9-2b28-4b34-ac03-73933472344d.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDcxMTQyMjIsIm5iZiI6MTcwNzExMzkyMiwicGF0aCI6Ii8yNjI5LzMwMTg0Mjk2MS1mNDlmZTdkOS0yYjI4LTRiMzQtYWMwMy03MzkzMzQ3MjM0NGQucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDIwNSUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDAyMDVUMDYxODQyWiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MmU3N2M0ZjE4NWZiMDFmMDJhYzQ5MmJmMzg2MzcwZDc0NDJiYmEzZDUxMjgzN2JjNmEyNmRmMTg4NjljM2I4OCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.s5TTDf46GPGWS9_8NEsL9Ijghxazvrw8s5aceN_yov4)
> 
> However, according to https://daisyui.com/docs/colors/ it's not possible to customize that?

You can follow the steps: 

In index.css

```

.btn-disabled {
@apply !text-white
}
```

remember to use ! before styles to make it work

**Reply by jaimeiniesta** [reply]
Thanks, but that's not compatible with themes.

**Comment by saadeghi** [comment, marked as answer]
By default, it's `neautral` color with 20% opacity. So if you change the neutral color it will also change (with 20% opacity)  

You can apply a different color using utility classes or from config:  
https://play.tailwindcss.com/PDO9MRQbNI?file=config

Let me know if you have a question.
